docs/architecture/modernize-desktop/windows-migration.md

```diff
@@ -0,0 +1,524 @@
+    <Identity Name="YOUR-APP-GUID "
+              Publisher="CN=YOUR COMPANY"
+              Version="1.x.x.x" />
+</Package>
+```
+
+You can check which APIs need a packaged application identity by inspecting if
+the type that contains the API is marked with the
+[DualApiPartition](xref:Windows.Foundation.Metadata.DualApiPartitionAttribute)
+attribute. If it is, you can call if from an unpackaged traditional desktop app.
+Otherwise, you must convert your classic desktop app to a UWP with the help
+of a packaging project.
+
+<https://docs.microsoft.com/windows/desktop/apiindex/uwp-apis-callable-from-a-classic-desktop-app>
+
+#### Benefits of packaging
+
+Besides giving you access to these APIs, you get some additional benefits by
+creating a Windows App package for your desktop application including:
+
+* **Streamlined deployment**. Apps have a great deployment experience ensuring
+  that users can confidently install an application and update it. If a user
+  chooses to uninstall the app, it's removed completely with no trace left
+  behind preventing the Windows rot problem.
+
+* **Automatic updates and licensing**. Your application can participate in the
+  Microsoft Store's built-in licensing and automatic update facilities.
+  Automatic update is a highly reliable and efficient mechanism, because only
+  the changed parts of files are downloaded.
+
+* **Increased reach and simplified monetization**. Maybe not your case but if
+  you choose to distribute your application through the Microsoft Store you
+  reach millions of Windows 10 users.
+
+* **Add UWP features**. You can add UWP features to your app's package at your
+  own pace.
+
+#### Prepare for packaging
+
+Before proceeding to package your desktop application, there are some points you
+have to address before starting the process. Your application must respect any
+of the Microsoft Store rules and policies and run in the UWP application model.
+For example, it has to run on the .NET Framework 4.6.2 or later and writes to
+the `HKEY_CURRENT_USER` registry hive and the AppData folders will be virtualized
+to a user-specific app-local location.
+
+The design goal for packaging is to separate the application state from system
+state while maintaining compatibility with other apps. Windows 10 accomplishes
+this goal by placing the application inside a UWP
+package. It detects and redirects some changes to the file system and registry
+at run time to fulfill the promise of a trusted and clean install and uninstall
+behavior of an application provided by packaging.
+
+Packages that you create for your desktop application are desktop-only,
+full-trust applications that aren't sandboxed, although there's lightweight
+virtualization applied to the app for writes to `HKCU` & `AppData`. This allows
+them to interact with other apps the same way classic desktop applications do.
+
+##### Installation
+
+App packages are installed under `C:\\Program Files\\WindowsApps\\package_name`,
```

Comment:
```suggestion
App packages are installed under *C:\\Program Files\\WindowsApps\\package_name*,
```

Reply:
We also commonly refer to the program files directory as %ProgramFiles%, since users can choose a different default drive.